### DISCUSSION TITLE
Why did Coqui's SaaS shut down?

### DISCUSSION THREAD

**Discussion by platform-kit** [coqui-ai/TTS#3367] [General]
See title.

**Comment by erogol** [comment]
We decided so and want to focus on what we do best open-source and model development.

**Reply by platform-kit** [reply]
OK, fair. Does that mean XTTS will be open for commercial use on its next release? Or are you and the team still pursuing monetization some other way?

**Reply by loganhart02** [reply]
we are selling commercial licenses for XTTS models
more info here https://twitter.com/coqui_ai/status/1730698693781619048

**Reply by platform-kit** [reply]
@loganhart02 @erogol  what happens to the XTTS license in light of the latest announcement that Coqui is shutting down?

**Reply by erogol** [reply]
stays as it is at least for now
